src/outputs/parthenon_hdf5.cpp

```diff
@@ -307,7 +307,85 @@ void PHDF5Output::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
       int meshblock_size[3] = {nx1, nx2, nx3};
       const hsize_t xDims[1] = {3};
       writeH5AI32("MeshBlockSize", meshblock_size,
-                  H5S::FromHIDCheck(H5Screate_simple(1, xDims, NULL)), myDSet);
+                  H5S::FromHIDCheck(H5Screate_simple(1, xDims, NULL)), myDSet);  
+    }
+
+    { // TODO(forrestglines) - should this data be confined to yt-Info
+      // Store yt-necessary attributes
+      H5S const localDSpace = H5S::FromHIDCheck(H5Screate(H5S_SCALAR));
+      H5D const ytInfo =
```

Comment:
I actually see no reason to tag this as "needed for yt." Who knows, other vis tools might find this output useful.

Reply:
After writing the prototype, I agree. It's a bit convoluted to wrap these into a separate dataset and should belong in the `Info` dataset with everything else. I think we should always write this data rather than create two classes of outputs - one that's readable by `yt` (or other codes) and one that's not